Riot/Modules/Room/TimelineCells/Styles/Plain/Cells/FileAttachment/Incoming/Clear/RoomIncomingAttachmentBubbleCell.m

```diff
@@ -27,12 +27,7 @@ - (void)customizeTableViewCellRendering
 {
     [super customizeTableViewCellRendering];
     
-<<<<<<< HEAD
-    self.userNameLabel.textColor = ThemeService.shared.theme.userNameColors[0];
-    self.messageTextView.tintColor = ThemeService.shared.theme.tintColor;
-=======
     [self updateUserNameColor];
```

Comment:
Should we apply colors in this method ?

Reply:
In fact, `MXKRoomBubbleTableViewCell+Riot`.`updateUserNameColor` method body is fully commented.

Reply:
You could keep your changes but you have to add the following code `self.userNameLabel.textColor = ThemeService.shared.theme.userNameColors[0];` in `updateUserNameColor` method

Reply:
Done